src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_PER_DAY = 7;
+  const LONG_PER_DISCONT = 50;
+  const SHORT_PER_DAY = 3;
+  const SHORT_PER_DISCONT = 20;
```

Comment:
The variable names `LONG_PER_DAY`, `LONG_PER_DISCONT`, `SHORT_PER_DAY`, and `SHORT_PER_DISCONT` are not very descriptive. Consider renaming them to something more meaningful, such as `LONG_TERM_THRESHOLD`, `LONG_TERM_DISCOUNT`, `SHORT_TERM_THRESHOLD`, and `SHORT_TERM_DISCOUNT` to improve code readability.